### PR TITLE
Hotfix branch

### PR DESCRIPTION
**Pull request by Joshua-Barnes03**
Branch is a hotfix, development should be mostly stable after this.

-removed client directory, moved files into login folder in src
-commented out broken files, leaving to backend team to resolve syntax errors in their files
-commented out body parser but needs to be removed
-changed scripts to fit new filenames

### CODE REVIEW

File: backend/server/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const expressSession = require('express-session')
 // commented out for now
 // const bodyParser = require('body-parser')
 
-//TODO: this throws a huge error because
+//TODO: this throws a huge error because mongoose is not hooked up yet, please resolve
```
Comment (smcgovern-proj) [inline backend/server/index.js]: can you be more specific here so they know exactly what they have to do

Comment (Joshua-Barnes03) [reply backend/server/index.js]: changed to become more clear, nice catch

**Review by clendon — COMMENTED** [review]
Runs on my end, minus the mongo connection.

**Review by smcgovern-proj — APPROVED** [review]

**Review by clendon — APPROVED** [review]
Everything works except mongo. But the server is running.

**Pull request merged**